### PR TITLE
test: doc: fix units in tx-size standardness test (s/vbytes/weight units)

### PR DESCRIPTION
**Pull request by theStack**
This small fixup PR is a late follow-up for #17947 (commit 4537ba5f21ad8afb705325cd8e15dd43877eb28f), where the wrong units has been used in the comments for the large tx composition.

### CODE REVIEW

**Review by tdb3 — APPROVED** [review]
ACK d1581c6048478cf70c5fb9ec5ebc178f16b376b8
Normally I would think this change could be included in a future change to `test_IsStandard`, but there is value in preventing confusion and not hoping that a future reviewer/author remembers to include this change.

File: src/test/transaction_tests.cpp [hunk 1]
```diff
@@ -904,15 +904,15 @@ BOOST_AUTO_TEST_CASE(test_IsStandard)
...
     t.vin.resize(2438); // size per input (empty scriptSig): 41 bytes
     t.vout[0].scriptPubKey = CScript() << OP_RETURN << std::vector<unsigned char>(19, 0); // output size: 30 bytes
-    // tx header:                12 bytes =>     48 vbytes
-    // 2438 inputs: 2438*41 = 99958 bytes => 399832 vbytes
-    //    1 output:              30 bytes =>    120 vbytes
-    //                      ===============================
-    //                                total: 400000 vbytes
+    // tx header:                12 bytes =>     48 weight units
+    // 2438 inputs: 2438*41 = 99958 bytes => 399832 weight units
+    //    1 output:              30 bytes =>    120 weight units
```
Comment (ismaelsadeeq) [inline src/test/transaction_tests.cpp:909]: Are these `bytes` values `vbytes`?

**Review by ismaelsadeeq — COMMENTED** [review]
ACK d1581c6048478cf70c5fb9ec5ebc178f16b376b8

C.I Failure is unrelated https://github.com/bitcoin/bitcoin/actions/runs/9436375635/job/25991001694?pr=30254

File: src/test/transaction_tests.cpp (hunk 1 above)
Comment (glozow) [reply src/test/transaction_tests.cpp:909]: Both AFAICT. Presumably the idea here is to show how weight units are calculated from bytes, so it doesn't make much sense to start from vbytes.

**Review by glozow — COMMENTED** [review]
ACK d1581c6048478cf70c5fb9ec5ebc178f16b376b8

functional test CI failure looks unrelated

**Pull request merged**